test/index.js

```diff
@@ -1,126 +1,122 @@
-const test = require('tape')
-const once = require('once')
-const { PassThrough, Transform } = require('readable-stream')
-const endOfStream = require('end-of-stream')
-const pump = require('pump')
-const ObjMultiplex = require('../index.js')
+/* eslint-disable node/no-unpublished-require */
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
This rule should be enabled, and all unused variables should either be deleted (preferred) or prefixed with `_`.